webapp/backend/src/handlers/api/videos.ts

```diff
@@ -0,0 +1,63 @@
+import { Request, Response, Router } from "express";
+import fs from "fs";
+import allVideos from "../../videoData";
+import path from "path";
+
+// Initialise router object
+const demoVideos = Router();
+
+// Metadata for all videos
+demoVideos.get("/", (req: Request, res: Response) => {
+  res.json(allVideos);
+});
+
+// Metadata for a single video
+demoVideos.get("/:id/metadata", (req: Request, res: Response) => {
+  const id = parseInt(req.params.id, 10);
+  res.json(allVideos[id]);
+});
+
+// Endpoint for a single video to stream
+demoVideos.get("/video/:id", (req: Request, res: Response) => {
```

Comment:
do some error checking of the id passed here. shouldn't implicitly trust that it is correct or safe